CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ Changelog for the gruf gem. This includes internal history before the gem was ma
 - Add script/e2e test for full e2e test 
 - Explicitly declare [json gem](https://rubygems.org/gems/json) dependency
 
+### 2.8.2
```

Comment:
We don't want to necessarily presume 2.8.2 will be the next thing to land. (In fact, it likely wont due to the other stuff in `master` - more likely be 2.9) I'd add this below the currently existing `Pending Release` items.